pages.ml/linux/brightnessctl.md

```diff
@@ -0,0 +1,24 @@
+# brightnessctl
+
+> മോണിറ്ററിന്റെ പ്രകാശം നിയന്ത്രിക്കാനുള്ള യൂട്ടിലിറ്റി.
```

Comment:
Can you update this line alone to better align with the [main page](https://github.com/tldr-pages/tldr/blob/main/pages/linux/brightnessctl.md).

Reply:
Updated